src/components/UI/Modal/pages/LoginModal.test.tsx

```diff
@@ -209,9 +213,11 @@ describe('LoginModal component', () => {
 				expect(
 					queryByText('We found environment variables with connection details for')
 				).not.toBeInTheDocument()
-				expect(getByText('Almost there!')).toBeInTheDocument()
+				expect(queryByText('Verification Code')).toBeInTheDocument()
 				expect(
-					getByText('Please go ahead and complete the login process in the opened browser')
+					getByText(
+						'After you login using the browser, you’ll be asked to enter the verification code above.'
```

Comment:
Please consider the following:
```suggestion
						'Upon logging in through your browser, you might receive a prompt to enter the verification code above.'
```